app/addons/activetasks/tests/activetasks.componentsSpec.react.jsx

```diff
@@ -70,6 +70,14 @@ define([
         window.confirm.restore && window.confirm.restore();
       });
 
+
+      it('it displays a message instead of an empty table, if there are undefined active tasks', function () {
+        spy = sinon.spy(Actions, 'setCollection');
```

Comment:
maybe I'm not fully getting it, but we call the method we just mocked? Would we not rather leave out this line and see what happens?